### ISSUE TITLE
Get multiple items in a single function call

### ISSUE THREAD

**Issue by SebastienMelki** [allegro/bigcache#324]
Hello, I was wondering if bigcache could support a getMulti(keys []string) function or if there was a technical limitation preventing it.

Thanks!

**Comment by janisz** [comment]
Hey, I think it's not a bug but enhancement. I thin it make sens to expose get/set for multiple items as it can boost performance as there will be only single lock. This may be useful to setup/backup the cache. 

@cristaloleg WDYT?

**Comment by SebastienMelki** [comment]
> Hey, I think it's not a bug but enhancement. I thin it make sens to expose get/set for multiple items as it can boost performance as there will be only single lock. This may be useful to setup/backup the cache.
> 
> @cristaloleg WDYT?

Yea sorry the bug label came by default and I wasn't able to remove it. I completely agree @janisz , if you can point me to what I should look at I can take a jab at implementing the multi approach.

**Comment by cristaloleg** [comment]
Yeah, sounds good. With a multiget we can increase throughput because we will take lock once.

**Comment by raeidish** [comment]
Is this available?

**Comment by cristaloleg** [comment]
@raeidish looks so
